lib/authentication/secure_storage/json_credential_manager.js

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2015-2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+const path = require('path');
+const Logger = require('../../logger');
+const fs = require('fs');
+const os = require('os');
+const Util = require('../../util');
+
+function JsonCredentialManager() {
+
+  this.userinfo = os.userInfo();
+
+  this.getTokenDir = function () {
+    let tokenDir = process.env.SF_TEMPORARY_CREDENTIAL_CACHE_DIR;
```

Comment:
should it be an environment variable or can we make it parameter configured via `snowflake.configure(...)`?

Reply:
I think that will be better for customers. The reason why I used it is the JDBC is using process.env.SF_TEMPORARY_CREDENTIAL_CACHE_DIR parameter. I will fix it.